src/Illuminate/Contracts/Pagination/Paginator.php

```diff
@@ -43,6 +43,15 @@ public function nextPageUrl();
      */
     public function previousPageUrl();
 
+    /**
+     * Get the URL for the first page.
+     *
+     * @author Stephen Damian <contact@devandweb.fr>
+     *
+     * @return string
+     */
+    public function firstPageUrl();
```

Comment:
Adding new methods to contracts on a minor/patch release is unfortunately a breaking change. Please remove this again from this PR.